aliBuild

```diff
@@ -237,10 +247,10 @@ class HttpRemoteSync:
       else:
         hashList = json.loads(urlopen(hashListUrl).read())
         pkgList = json.loads(urlopen(pkgListUrl).read())
-    except URLError, e:
+    except URLError as e:
```

Comment:
I think the only way to support the different syntaxes is to have separate files that are loaded by "globally compatible" (i.e. super simple/trivial) executable scripts (aliBuild, etc).

I think it would be possible to use the 2to3 script to minimize (human) code duplication. I don't know if it would be easy to apply to whole modules (such as the local yaml module).